openstates/in/__init__.py

```diff
@@ -71,9 +67,16 @@
 )
 
 def session_list():
-    from billy.scrape.utils import url_xpath
-    # cool URL bro
-    return url_xpath('http://www.in.gov/legislative/2414.htm', '//h3/text()')
+    import requests
+    import os
+    apikey = os.environ['INDIANA_API_KEY']
```

Comment:
re API_KEY, this might be working somewhere because the scraper wasn't failing before. Although maybe it just never called anything that needed that. Tossing this one to @paultag